src/DS2431.h

```diff
@@ -0,0 +1,28 @@
+// 0x2D  1Kb 1-Wire EEPROM
+// tested on buspirate and two different real 1-wire masters
+
+#ifndef ONEWIRE_DS2431_H
+#define ONEWIRE_DS2431_H
+#define CRC_INIT 0xCFB5
```

Comment:
what do you mean by that? yes it is not in the standard, but all except one c-compiler understand this pragma. Should be ok for quite a while now